.github/workflows/ci.yml

```diff
@@ -191,6 +277,37 @@ jobs:
         echo "LLVM_HASH=$LLVM_HASH" >> $GITHUB_ENV
     - uses: nelonoel/branch-name@v1.0.1
 
+    - name: Save PR Info on Windows systems
```

Comment:
Need to determine how to get CLING_HASH AND LLVM_HASH on Windows. Have idea on how to solve this. Will test tomorrow.

Reply:
maybe %LLVM_HASH%